include/openssl/err.h

```diff
@@ -84,12 +84,12 @@ typedef struct err_state_st {
 # define ERR_LIB_COMP            41
 # define ERR_LIB_ECDSA           42
 # define ERR_LIB_ECDH            43
-# define ERR_LIB_STORE           44
```

Comment:
Not sure, but who do you explain that some defines (like 15-lines or more above) are commented ?
I have not yet  dig the pelr/mkerr code.